freebasics/settings/base.py

```diff
@@ -212,6 +212,19 @@
 
 ENABLE_SSO = False
 
+CUSTOM_CSS_BASE_BACKGROUND_COLOR = environ.get(
+    'CUSTOM_BODY_BACKGROUND_COLOR', "#ffffff")
```

Comment:
And same for the rest below - they should match the env variables we're creating in `mc2-freebasics`